.gitignore

```diff
@@ -1,5 +1,6 @@
 ._*
 .DS_Store
+.idea
```

Comment:
Should this be here?

Reply:
I can remove it, I use PHPStorm instead of vscode